docs/src/main/paradox/project/migration-guides.md

```diff
@@ -10,16 +10,29 @@ please [migrate to Akka 2.6](https://doc.akka.io/docs/akka/current/project/migra
 
 This is just stub documentation. It will be improved.
 
-* for Pekko jar dependencies, the groupId is "org.apache.pekko" instead of "com.typesafe.akka"
-* the jar names start with "pekko" instead of "akka" - e.g. pekko-actor_2.13.jar instead of akka-actor_2.13.jar 
+In your dependency management files:
+* Update dependency Akka version to the latest Pekko version.
```

Comment:
Rather than latest Pekko version I would instead say latest version of Pekko 1.0.x (which we claim to be broadly compatible with latest Akka BSL release).

Reply:
```suggestion
* Update dependency Akka version to the latest version of Pekko 1.0.x.
```